### PR TITLE
Add Sentry config to app pages

### PR DESCRIPTION
**Pull request by redbugz**
Sets the following properties on `SERVER_DATA`:

- sentryDSN
- sentryDisabled
- sentryRelease
- targetEnv
 
`sentryRelease` is formatted like: 
`{appName}@{EFBuildNum}:{ShortGitSHAHash}`, i.e.
`frontier-app-react@172:6386b217`

### CODE REVIEW

File: packages/react-scripts/layout/views/layout.ejs [hunk 1]
```diff
@@ -42,9 +42,11 @@
 
     <script>
       window.SERVER_DATA = {};
-      <%- typeof cacheKey !== 'undefined' ? `SERVER_DATA.cacheKey = '${cacheKey}';` : ''; %>
-      SERVER_DATA.appPath = "<%- typeof appPathOverride !== 'undefined' ? appPathOverride : appPath('') %>";
+      <%= typeof cacheKey !== 'undefined' ? `SERVER_DATA.cacheKey = '${cacheKey}';` : ''; %>
+      SERVER_DATA.appPath = "<%= typeof appPathOverride !== 'undefined' ? appPathOverride : appPath('') %>";
+      SERVER_DATA.targetEnv = "<%= process.env.TARGET_ENV %>""
```
Comment (joeycozza) [inline packages/react-scripts/layout/views/layout.ejs]: does this right here mean that we'll know targetEnv on the client side? Could I use this in the Link component?

Comment (redbugz) [reply packages/react-scripts/layout/views/layout.ejs]: Yes, but why would the Link component care which targetEnv you are?

Comment (redbugz) [reply packages/react-scripts/layout/views/layout.ejs]: And it reminds me I need to add some warnings to the docs to use this value carefully

Comment (joeycozza) [reply packages/react-scripts/layout/views/layout.ejs]: when developing locally, we decided that if the Link component would take you to a different app within familysearch, that we would literally just redirect you to the app depending on the target env.

Comment (redbugz) [reply packages/react-scripts/layout/views/layout.ejs]: oh ok, just for local dev. That makes sense

**Comment by redbugz** [comment]
Travis builds completed and passed, but for some reason is not updating the PR checks.

**Pull request merged**